draft-ietf-mls-protocol.md

```diff
@@ -260,48 +262,48 @@ This protocol is designed to execute in the context of a Messaging Service (MS)
 as described in [I-D.ietf-mls-architecture].  In particular, we assume
 the MS provides the following services:
 
-* A long-term identity key provider which allows participants to authenticate
+* A long-term identity key provider which allows clients to authenticate
   protocol messages in a group. These keys MUST be kept for the lifetime of the
-  group as there is no mechanism in the protocol for changing a participant's
+  group as there is no mechanism in the protocol for changing a client's
   identity key.
 
 * A broadcast channel, for each group, which will relay a message to all members
   of a group.  For the most part, we assume that this channel delivers messages
   in the same order to all participants.  (See {{sequencing}} for further
   considerations.)
 
-* A directory to which participants can publish initialization keys, and from which
-  participant can download initialization keys for other participants.
+* A directory to which clients can publish initialization keys and download
+  initialization keys for other participants.
 
 
 # Protocol Overview
 
-The goal of this protocol is to allow a group of participants to exchange confidential and
-authenticated messages. It does so by deriving a sequence of secrets and keys known only to group members. Those
+The goal of this protocol is to allow a group of clients, members of a group, to exchange confidential and
```

Comment:
This reads awkwardly.  Suggest deleting ", members of a group,"

Reply:
Fixed.